website/content/docs/v0.4/Guides/first-cluster.md

```diff
@@ -28,8 +28,16 @@ kubectl patch deploy -n sidero-system sidero-controller-manager --type='json' -p
 
 #### Update Environment
 
-The metadata server's information needs to be updated in the default environment.
-Edit the environment with `kubectl edit environment default` and update the `talos.config` kernel arg with the IP of one of the management plane nodes (or the DNS entry you created).
+The metadata server's information might need to be updated in the default environment.
```

Comment:
is this the metadata server or the sidero controller?

Reply:
Hmm, looks like I can probably remove whole section there